transformers/events/pot_file/vow/repository.go

```diff
@@ -0,0 +1,18 @@
+package vow
+
+import (
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Repository struct {
+	db *postgres.DB
+}
+
+func (repository Repository) Create(models []event.InsertionModel) error {
+	return event.Create(models, repository.db)
+}
+
+func (repository *Repository) SetDB(db *postgres.DB) {
+	repository.db = db
+}
```

Comment:
@rmulhol Does this really work after the removal of the transformer-specific repository from vdb core? :confused: